MANIFEST.in

```diff
@@ -2,20 +2,21 @@
 include CHANGES COPYING COPYING.LESSER
 
 # Files from setup.py package_data that are not automatically added to source distributions
-recursive-include lib/iris/tests/results *.cml *.cdl *.txt *.xml *.json
+recursive-exclude lib/iris/tests *
```

Comment:
How does this relate to #3835?

Reply:
As a holding position, I think it's fine to not bundle the tests with anything we package, regardless of where the tests live at the moment.

The tests refactor and adoption of `pytest` is super high on my hit list... so that's all coming soon :+1:

Reply:
I suppose we can feel more free to change the location with less worry if we already know we're not distributing the tests.

Reply:
Yeah, I think so... well that's my take on it :+1:

Reply:
So does this close #3416?

Reply:
@rcomer I think it touches on #3416, but doesn't address my original intent of that issue i.e., we're explicitly not including or nuking the tests when packaging for conda and PyPI which gives the same vendored result.

However, #3416 is questioning where the tests actually live in the repo and codebase i.e., if the tests didn't live under `lib/iris` then we wouldn't need to not include them or delete them before packaging. If you get my drift...

Reply:
It's preferable that sdists have tests. For wheels, that's a bit more optional.